cmd/secrets-provider/main.go

```diff
@@ -77,16 +77,10 @@ func main() {
 
 func provideSecretsToTarget(authn *authenticator.Authenticator, provideConjurSecrets secrets.ProvideConjurSecrets, accessToken *memory.AccessToken) error {
 	log.Info(fmt.Sprintf(messages.CSPFK001I, authn.Config.Username))
-	authnResp, err := authn.Authenticate()
-	if err != nil {
+	if err := authn.Authenticate(); err != nil {
```

Comment:
can we split this into 2 lines?

Reply:
Sure! Better to keep it consistent with the rest of the file.